cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -102,6 +108,9 @@ namespace detail {
  */
 class alignas(16) column_device_view_base {
  public:
+  // TODO: merge this offsets column index with `strings_column_view::offsets_column_index`
```

Comment:
Is this TODO in scope for this PR? Why/why not?

Reply:
It's a temporary repetition of the string children index, it doesn't affect any existing behaviour. I'll create a follow-up PR to move the offsets to a header that jitify can also compile. I intend to keep the PR short.

Reply:
Also, provided the offsets and data representation are stable and unlikely to change, I think it's okay. I'ts only a nice-to-have.